actix-web/hello-world/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+use actix_web::web::{resource, ServiceConfig};
+use shuttle_service::ShuttleActixWeb;
+
+async fn hello_world() -> &'static str {
+    "Hello World!"
+}
+
+#[shuttle_service::main]
+async fn actix_web(
+) -> ShuttleActixWeb<impl FnOnce(&mut ServiceConfig) + Sync + Send + Copy + Clone + 'static> {
+    let h = hello_world;
```

Comment:
Will this be needed for all routes? Ie assigning the pointer to a variable first?

Reply:
No, it's not required. Actually it was part of my experiments with closures. Fix pushed